Update Twitter Section | Right Click to Delete Markers | New Config Format

RedSparr0w opened the pull request:
Twitter Section:
- Added Custom Tags
- Min Time Between Tweets
- Added button to show tweet rather than displaying below editing area

Markers:
- Right click on marker to remove.

_will eventually make a menu to be able to delete marker or swap marker to a different "area name" once i have enough time to figure it out XD_

RedSparr0w committed:
Update Twitter Section

RedSparr0w committed:
right click circle to delete

RedSparr0w committed:
Remove Pokevision references

Removed Pokevision due to site shutdown

bunzosteele commented:
I'll test this out an merge this evening. I've been holding off really announcing this on reddit/wherever until you felt it was feature complete enough to be ready. let me know, and I'll start packing it up with the normal pokewatch download, and either I can post about it, or you can.

RedSparr0w committed:
Updated for new multiple account config

Still need to add each areas personal Exclude/Include List and Global
Exclude/Include/Priority List

RedSparr0w committed:
Update Scan Times

Scan time = Markers * Scantime(10) / TotalAccounts

RedSparr0w committed:
Add ability to adjust spacing for "Mass Mark Area"

added range input to adjust spread of "mass mark area" in order to get
tighter markers in your area

RedSparr0w committed:
Tidy Up Code

Tidy up javascript code a little bit and make it more organised

RedSparr0w commented:
@bunzosteele It should generate the new config format correctly now [=
Doesn't create the excludes, includes though. not sure how i'm going to go about that yet.
